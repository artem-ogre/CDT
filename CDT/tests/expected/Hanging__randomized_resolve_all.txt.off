OFF
10 15 0
462.6772156241689 153.493424163765 0
1512.322784375831 153.493424163765 0
987.5 1062.51315167247 0
725 415 0
855 390 0
945 455 0
1100 373 0
1215 410 0
1250 510 0
943 540 0
3 5 7 9
3 1 2 8
3 2 0 3
3 2 3 9
3 3 7 5
3 2 9 8
3 4 0 6
3 9 7 8
3 7 1 8
3 7 3 4
3 3 0 4
3 7 4 6
3 3 5 9
3 1 6 0
3 1 7 6
