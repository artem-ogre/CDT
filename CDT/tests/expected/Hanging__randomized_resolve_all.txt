15
0 1 6   4294967295 5 3
0 3 2   2 7 4294967295
0 4 3   3 9 1
0 6 4   0 12 2
1 2 8   4294967295 8 6
1 7 6   6 12 0
1 8 7   4 14 5
2 3 9   1 10 8
2 9 8   7 14 4
3 4 7   2 12 11
3 5 9   11 13 7
3 7 5   9 13 10
4 6 7   3 5 9
5 7 9   11 14 10
7 8 9   6 8 13

1
3 7

0

0
